sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1087,6 +1093,7 @@ public partial class HttpPipelineTransportOptions
     {
         public HttpPipelineTransportOptions() { }
         public System.Collections.Generic.IList<System.Security.Cryptography.X509Certificates.X509Certificate2> ClientCertificates { get { throw null; } }
+        public bool IsClientRedirectEnabled { get { throw null; } set { } }
```

Comment:
I'm interested in [this comment] @JoshLove-msft made, and its implication that we might be able to reduce added API surface area.  Can you help me understand why we need this flag in both the outer container and separately on the transport options?

Reply:
Containers is the pattern for ClientOptions , so I don't think we should add a top level property there. I suppose we could add a container in the transport options, but I don't know if that makes things better.

Reply:
Why is it that we also need it on the transport options?  If we have the same option in two places, what happens if one is set to true and one is set to false?